pkg/commands/build/build.go

```diff
@@ -48,11 +59,29 @@ var createInlinerFn = func(pbr files.FileObjReader) *build.Inliner {
 	return build.NewInlinerWithScheme(files.FileScheme, pbr)
 }
 
-func run(ctx context.Context, o *options, brw cmdlib.BundleReaderWriter, gopt *cmdlib.GlobalOptions) error {
+func run(ctx context.Context, o *options, brw cmdlib.BundleReaderWriter, rw files.FileReaderWriter, gopt *cmdlib.GlobalOptions) error {
 	bw, err := brw.ReadBundleData(ctx, gopt)
 	if err != nil {
 		return fmt.Errorf("error reading bundle contents: %v", err)
 	}
 
+	//TODO(jbelamaric): allow multiple files
+	// the bundle now contains components which may include PatchTemplateBuilder objects
+	// that we need to build into PatchTemplates
+        optFiles := []string{}
```

Comment:
formatting's a bit weird here

Reply:
i just removed the TODO, it's already captured elsewhere